src/styles/blocks/button.scss

```diff
@@ -0,0 +1,27 @@
+.button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: 280px;
+  height: 60px;
+  min-height: 60px;
+  box-sizing: border-box;
+  background-color: $gray1;
+  border: 1px solid $gray1;
+
+  cursor: pointer;
+
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 130%;
+  text-align: center;
+  letter-spacing: 0.15em;
+  text-transform: uppercase;
+  color: $gray6;
+
+  text-decoration: none;
+
+  @include hover(background-color, #E4DFD3);
```

Comment:
it won't work in this way:
![image](https://user-images.githubusercontent.com/86923089/191072369-f7e61057-6678-456c-af3c-bcda79fd5a93.png)

I know that it was already written mixin (it works fine with 1 prop), but it'll work better if in this case, you move `transition` directly to `.button`